packages/sdk-react/src/core/queries/useCounterpart.ts

```diff
@@ -441,6 +441,24 @@ export const useDeleteCounterpartContact = () => {
   );
 };
 
+export const useMakeCounterpartContactDefault = () => {
+  const queryClient = useQueryClient();
+  const { i18n } = useLingui();
+  const { api } = useMoniteContext();
+
+  return api.counterparts.postCounterpartsIdContactsIdMakeDefault.useMutation(
+    undefined,
+    {
+      onSuccess: async () => {
+        await api.counterparts.getCounterpartsIdContacts.invalidateQueries(
```

Comment:
Btw, please, always invalidate lists too to prevent any future problems.

Reply:
Could you please provide an example to clarify what I might be missing? I am having difficulty understanding the current problems

Reply:
Clarified in [the previous comment](https://github.com/team-monite/monite-sdk/pull/225#discussion_r1707840131).